cfgov/apache/conf.d/wsgi.conf.venv

```diff
@@ -1,3 +1,7 @@
+# Our classic mod_wsgi deployment will move this file into place if we're
+# expecting to use mod_wsgi.
+# TODO: Remove when the application is containerized or moved to Gunicorn
```

Comment:
```suggestion
# TODO: Remove when the application is containerized or moved to Gunicorn.
```

Reply:
Added in [37ef717](https://github.com/cfpb/consumerfinance.gov/pull/8676/commits/37ef71714dec0de4c40724d89013a9ea9bc2b28b)